### PR TITLE
Resource server integration tests

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
Adding support to run TCK based integration tests for jwt-verifier resource server flow

Also updated the tck jar to the latest version

### CODE REVIEW

File: packages/jwt-verifier/package.json [hunk 1]
```diff
@@ -21,6 +29,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "cors": "^2.8.4",
+    "express": "^4.15.4",
```
Comment (lboyette-okta) [inline packages/jwt-verifier/package.json]: these should be dev-dependencies

File: packages/jwt-verifier/test/integration-test/resource-server.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  const accessToken = match[1];
+
+  return oktaJwtVerifier.verifyAccessToken(accessToken)
+    .then((jwt) => {
+      req.jwt = jwt;
+      next();
+    })
+    .catch((err) => {
+      const header = 'Bearer realm=resource, error="unauthorized", error_description="' + err.message + '"';
```
Comment (lboyette-okta) [inline packages/jwt-verifier/test/integration-test/resource-server.js]: nit: use template string for templates when possible:

```typescript
const header = `Bearer realm=resource, error="unauthorized", error_description="${err.message}"`;
```

File: packages/jwt-verifier/test/integration-test/resource-server.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ */
+app.use(cors());
+
+/**
+ * An example route that requires a valid access token for authentication, it
+ * will display a boring message to the subject if the middleware successfully
+ * validated the token.
+ */
+app.get('/', authenticationRequired, (req, res) => {
+  res.send('The message of the day is boring: ' + req.jwt.claims.sub)
```
Comment (lboyette-okta) [inline packages/jwt-verifier/test/integration-test/resource-server.js]: nit: template string

```typescript
res.send(`The message of the day is boring: ${req.jwt.claims.sub}`)
```

File: packages/jwt-verifier/test/integration-test/resource-server.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ */
+app.get('/api/messages', authenticationRequired, (req, res) => {
+  res.json([{
+    message: 'Hello, word!'
+  }]);
+});
+
+app.listen(8080, () => {
+  console.log('Server Ready on port 8080');
+});
```
Comment (lboyette-okta) [inline packages/jwt-verifier/test/integration-test/resource-server.js]: nit: new line

File: packages/jwt-verifier/test/integration-test/resource-server.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
+const express = require('express');
```
Comment (lboyette-okta) [inline packages/jwt-verifier/test/integration-test/resource-server.js]: we should run `npm run banners` in the root of `okta-oidc-js` to ensure all of our js has a license banner

File: packages/jwt-verifier/test/integration-test/resources/testRunner.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    enabled: false
+
+  code-flow-local-validation:
+    enabled: false
+
+  code-flow-remote-validation:
+    enabled: false
+
+  implicit-flow-remote-validation:
+    enabled: false
```
Comment (robertjd) [inline packages/jwt-verifier/test/integration-test/resources/testRunner.yml]: newline

File: packages/jwt-verifier/package.json [hunk 7]
```diff
@@ -5,8 +5,16 @@
   "repository": "https://github.com/okta/okta-oidc-js",
   "homepage": "https://github.com/okta/okta-oidc-js/tree/master/packages/jwt-verifier",
   "main": "lib.js",
+  "config": {
+    "tck": {
+      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.1-SNAPSHOT&e=jar&c=shaded",
+      "setup_env": "export ISSUER=http://localhost:9090/oauth2/default export CLIENT_ID=OOICU812 export CLIENT_SECRET=VERY_SECRET",
```
Comment (jmelberg-okta) [inline packages/jwt-verifier/package.json]: While we are here, should probably use the [cross-env](https://www.npmjs.com/package/cross-env) package to make this compatible with other operating systems

File: packages/jwt-verifier/test/integration-test/resource-server.js (hunk 5 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/test/integration-test/resource-server.js]: Done

File: packages/jwt-verifier/test/integration-test/resource-server.js (hunk 2 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/test/integration-test/resource-server.js]: Done

File: packages/jwt-verifier/test/integration-test/resource-server.js (hunk 3 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/test/integration-test/resource-server.js]: Done

File: packages/jwt-verifier/test/integration-test/resource-server.js (hunk 4 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/test/integration-test/resource-server.js]: Done

File: packages/jwt-verifier/test/integration-test/resources/testRunner.yml (hunk 6 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/test/integration-test/resources/testRunner.yml]: Done

File: packages/jwt-verifier/package.json (hunk 7 above)
Comment (vijetmahabaleshwar-okta) [reply packages/jwt-verifier/package.json]: Added

**Pull request merged**